docs/testing/testplan.md

```diff
@@ -0,0 +1,355 @@
+
+### Prerequisites
+
+Before you start testing, sign-up for a wallet:
+
+1. The sign-up process depends on your digital wallet provider. Learn more from the [Dependencies](#Dependencies) section.
+2. Your wallet provider will likely need to verify your identity (info: this is mandated by the laws of each country, or jurisdiction).
+3. The identity verification process depends on your wallet provider, and your location (info: this can be within hours or days).
+4. Once your wallet provider completes all sign-up and verification processes, you should be able to:  
+   a. Setup and manage your wallet accounts, currencies, balances etc.  
+   b. Obtain the wallet address or payment pointer for your digital wallet.
+
+### Test Data
+
+This table describes the different ways that you can set up a website that is monetized, and a website that isn’t monetized, so that you can test the Web Monetization extension.
+
+| Web monetized websites                                                                                                                                                                                                                      | Non monetized websites    |
+| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :------------------------ |
+| One valid monetization link tag                                                                                                                                                                                                             | No monetization link tags |
+| Multiple valid monetization link tags with wallet addresses: <ul><li>from the same wallet providers</li><li>from different wallet providers</li><li>with different currencies</li><li>with some link tags enabled & some disabled</li></ul> | All disabled link tags    |
+
+### Test Playground
+
+Use the [Web Monetization playground](https://webmonetization.org/play/) to quickly and seamlessly create the conditions of a monetized website, to use only for testing. When you add wallet addresses or payment pointers to the playground, the playground becomes a monetized website that you can use the Web Monetization extension to pay
+
+On the playground, you can use real money from a real wallet, or you can use “play” money from a “test wallet”.
+
+**Important**
+
+1. The playground itself is not web monetized.
+2. The playground becomes monetized when you add one or more receiving wallet addresses to it.
+3. If your Web Monetization extension is connected to a real wallet, then your extension will facilitate payments from your real wallet (i.e. real money).
+4. Similarly, if your extension is connected to a play wallet (i.e. rafiki.money), then the extension facilitates payments using play money.
```

Comment:
What happens if I add one play wallet address and one real wallet address to the playground?

Reply:
It depends on:
- the wallet that your extension is connected to;
- which of one those two your wallet is peered with...best case scenario, your wallet would be peered with only one, and so 50% of any payment the extension attempts can be successful (ref: Test ID 9)